pinot-spi/src/main/java/org/apache/pinot/spi/config/table/IndexingConfig.java

```diff
@@ -27,6 +27,7 @@
 
 public class IndexingConfig extends BaseJsonConfig {
   private List<String> _invertedIndexColumns;
+  private List<String> _rangeIndexColumns;
```

Comment:
We should not make a change to IndexingConfig. Let's start using FieldConfig.
IndexLoadingConfig can then be derived from FieldConfig

Reply:
Similar to how we do it for text index.

Reply:
I agree. will have to do this as part of another PR.

Reply:
Let's please add a TODO